.travis.yml

```diff
@@ -1,47 +1,61 @@
+# Ubuntu 14.04 Trusty support
+sudo: required
```

Comment:
Just a comment: Jobs requiring `sudo` [are slower](https://docs.travis-ci.com/user/ci-environment/) (see `Boot Time`) but otherwise you can not use `apt-get install`.